<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>networkWarnDlg</class>
 <widget class="QDialog" name="networkWarnDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="CLabel" name="labelTip">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>660</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>当前非Wifi网络，是否继续 ？</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="CLabel" name="labelContinue">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>660</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>继  续</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="CLabel" name="labelNeverWarn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>660</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>不再提示</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="CLabel" name="labelExit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>530</y>
     <width>660</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>退  出</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CLabel</class>
   <extends>QLabel</extends>
   <header location="global">label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
